.github/workflows/windows-build.yml

```diff
@@ -34,10 +34,17 @@ jobs:
         shell: cmd /C CALL {0}
         run: |
           call conda activate pyos-build
+          call conda list
```

Comment:
This debugging step is probably not needed

Reply:
Fixed